css/site.styl

```diff
@@ -31,49 +31,86 @@ normalize()
 // GRID
 
 $cols = 12
-$col = 90px
-$gutter = 16px
+$gutter = 24px
 
 .container
   margin: 0 auto
-  width: ($col * $cols)+($gutter * $cols)
+  width: 96%
 
 .row
-  clearfix()
-
-.col
-  float: left
-  margin-left: ($gutter/2)
-  margin-right: ($gutter/2)
-  min-height: 1px
+  box-sizing: border-box
+  display: flex
+  flex: 0 1 auto
+  flex-direction: row
+  flex-wrap: wrap
+  padding-top: ($gutter / 2)
+  &.reverse
+    flex-direction: row-reverse
 
-for col in 1..$cols
-  .col{col}
-    width: ( col * $col ) + ( ( col - 1 ) * $gutter )
+// FLEX POSITIONING
 
-    // FLUID COLS
+.flex-col
+  box-sizing: border-box
+  display: flex
+  flex-direction: column
+  &.reverse
+    flex-direction: column-reverse
 
-.percent-col
-  float: left
+.flex-row
   box-sizing: border-box
+  display: flex
+  flex-direction: row
+  flex-wrap: wrap
+  &.reverse
+    flex-direction: row-reverse
+
+.justify-start
+  justify-content: flex-start
+
+.justify-center
+  justify-content: center
 
-$pcols = 10
+.justify-end
+  justify-content: flex-end
 
-for pcol in 1..$pcols
-  .s-into-{pcol}
-    width: ( 100% / pcol )
+.align-start
+  align-items: flex-start
 
-@media screen and (min-width: 601px)
+.align-center
+  align-items: center
 
-  for pcol in 1..$pcols
-    .m-into-{pcol}
-      width: ( 100% / pcol )
+.align-end
+  align-items: flex-end
 
-@media screen and (min-width: 993px)
+.align-content-start
+  align-content: flex-start
 
-  for pcol in 1..$pcols
-    .l-into-{pcol}
-      width: ( 100% / pcol )
+.align-content-center
+  align-content: center
+
+.align-content-end
+  align-content: flex-end
+
+// COLUMN SIZES & OFFSETS
+
+.col
+  flex: 1 0 auto
+  box-sizing: border-box
+  max-width: 100%
+  margin-bottom: ($gutter / 2)
```

Comment:
yh i kno but i perfer to build without auto top or bottom margins but i'll merge and we see how it goes. i like the desktop solution i think